i18n/.js/src/main/scala/io/udash/i18n/bindings/TranslationBindingHelper.scala

```diff
@@ -0,0 +1,48 @@
+package io.udash.i18n.bindings
+
+import io.udash.logging.CrossLogging
+import org.scalajs.dom._
+import org.scalajs.dom.raw.Node
+
+private[bindings] trait TranslationBindingHelper extends CrossLogging {
+  import TranslationBindingHelper._
+
+  implicit def String2Node(str: String): Node =
```

Comment:
Implicit conversions on very commonly used types are considered a bad practice and tend to break when least expected. Especially in this case, since you're shadowing the default scalatags conversions for rendering strings.